packages/environment/src/deployments/arbitrum.ts

```diff
@@ -9,7 +9,9 @@ export default defineDeployment<Deployment.ARBITRUM>({
   externalContracts: {
     aaveV2IncentivesController: "0x0000000000000000000000000000000000000000",
     aaveV2LendingPoolProvider: "0x0000000000000000000000000000000000000000",
+    aaveV3IncentivesController: "0x929ec64c34a17401f460460d4b9390518e5b473e",
```

Comment:
aave is using both Incentives, and Rewards name, but I named it in the protocol the name that contract name has on the blockchain, that is `RewardsController`. Could we use the same name across PR?

Reply:
Yes, we can do that. I'll change it.

Reply:
thx